common/tier4_autoware_utils/src/vehicle/vehicle_state_checker.cpp

```diff
@@ -36,31 +36,31 @@ bool VehicleStopChecker::isVehicleStopped(const double stop_duration = 0.0) cons
     return false;
   }
 
+  constexpr double stop_velocity = 1e-3;
   const auto now = clock_->now();
-  const auto time_buffer = now - twist_buffer_.back().header.stamp;
-  if (time_buffer.seconds() < stop_duration) {
+
+  const auto time_buffer_front = now - twist_buffer_.back().header.stamp;
```

Comment:
This naming (front = back) is very confusing. It can be removed because the next check can be removed.